tests/scanners/test.css.js

```diff
@@ -9,48 +9,44 @@ import { ignorePrivateFunctions } from 'utils';
 import { getRuleFiles, metadataPassCheck, validMetadata } from '../helpers';
 
 
-describe('CSSScanner', () => {
+describe('CSSScanner', async () => {
   it('should report a proper scanner name', () => {
     expect(CSSScanner.scannerName).toEqual('css');
   });
 
-  it('should add CSS_SYNTAX_ERROR with invalid css', () => {
+  it('should add CSS_SYNTAX_ERROR with invalid css', async () => {
     const code = '#something {';
     const cssScanner = new CSSScanner(code, 'fakeFile.css');
 
-    return cssScanner.scan()
-      .then(({ linterMessages }) => {
-        expect(linterMessages.length).toEqual(1);
-        expect(linterMessages[0].code).toEqual(messages.CSS_SYNTAX_ERROR.code);
-        expect(linterMessages[0].type).toEqual(VALIDATION_WARNING);
-        expect(linterMessages[0].message).toEqual('Unclosed block');
-        expect(linterMessages[0].line).toEqual(1);
-        expect(linterMessages[0].column).toEqual(1);
-        expect(linterMessages[0].file).toEqual('fakeFile.css');
-      });
+    const { linterMessages } = await cssScanner.scan();
+    expect(linterMessages.length).toEqual(1);
+    expect(linterMessages[0].code).toEqual(messages.CSS_SYNTAX_ERROR.code);
+    expect(linterMessages[0].type).toEqual(VALIDATION_WARNING);
+    expect(linterMessages[0].message).toEqual('Unclosed block');
+    expect(linterMessages[0].line).toEqual(1);
+    expect(linterMessages[0].column).toEqual(1);
+    expect(linterMessages[0].file).toEqual('fakeFile.css');
   });
 
-  it('should pass metadata to rules', () => {
+  it('should pass metadata to rules', async () => {
     const code = oneLine`/* whatever code */
       #myName { position: relative; }
       .myClass { background: #000; }`;
-    const fakeRules = { metadataPassCheck: () => {} };
+    const fakeRules = { metadataPassedCheck: () => {} };
 
     // This rule calls assert.fail() if no metadata is passed to it.
-    sinon.stub(fakeRules, 'metadataPassCheck').callsFake(metadataPassCheck);
+    sinon.stub(fakeRules, 'metadataPassedCheck').callsFake(metadataPassCheck);
```

Comment:
@rpl I changed metadataPassCheck to metadataPassedCheck because we have similar test in test.base.js lines 63-74 and there const fakeRules = { metadataPassedCheck: () => {} }; I thought it should be the same here so it constant from file to file.

Reply:
Ah, I didn't notice the other one.

Yeah, having a consistent name across the test is definitely better.
Fine with me, then.